templates/quiz/categorize.html

```diff
@@ -6,8 +6,8 @@ <h2>Categorize difficulties</h2>
 <tr class="difficulty_{{q.difficulty}}">
     <td>
         <a id="question_{{q.pk}}"/>
-        <pre class="sh_cpp">
-{{q.question}}
+        <pre>
+            <code class="cpp hljs">{{q.question}}</code>
```

Comment:
The added `code` inside the `pre` results in a bit more margin than we used to have, but I guess the site could use some whitespace anyway! :)

Reply:
The reason why we have both `<pre>` and `<code>` there is that Highlight.js by default treats this combination as code blocks to process (but [that could be customized](https://github.com/isagalaev/highlight.js#custom-initialization)).

I could add something like a "code-container" class on the outer `<pre>` there, and fiddle with the margins in CSS on that class, what do you think?

Reply:
Nah, don't bother. I'm planning to adjust the margins a bit in other places too.